modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,36 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return object autoloaded data - autoload size, entries, option and its names.
```

Comment:
See my other comment, if we return the results we also need to change this documentation since that means the return type is different (an `array`).